### PR TITLE
Purchase detail

### PR DESCRIPTION
**Pull request by PechBunsinh**
-generate Payment controller
-add and install gem file for bootstrap
-design purchase details list HTML and CSS

### CODE REVIEW

File: pj_movie/app/assets/javascripts/admin/customize.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+window.onload = function () {
```
Comment (N310) [inline pj_movie/app/assets/javascripts/admin/customize.js]: I think this file name is `purchase_detail.js` better.

File: pj_movie/app/assets/javascripts/admin/script.js [hunk 2]
```diff
@@ -0,0 +1,899 @@
+(function(){function qa(h,p){h.prototype=eb(p.prototype);h.prototype.constructor=h;h.base=p.prototype}function eb(h){function p(){}p.prototype=h;return new p}function Ya(h,p,D){"millisecond"===D?h.setMilliseconds(h.getMilliseconds()+1*p):"second"===D?h.setSeconds(h.getSeconds()+1*p):"minute"===D?h.setMinutes(h.getMinutes()+1*p):"hour"===D?h.setHours(h.getHours()+1*p):"day"===D?h.setDate(h.getDate()+1*p):"week"===D?h.setDate(h.getDate()+7*p):"month"===D?h.setMonth(h.getMonth()+1*p):"year"===D&&h.setFullYear(h.getFullYear()+
```
Comment (N310) [inline pj_movie/app/assets/javascripts/admin/script.js]: Is this file `jquery.canvasjs.min.js` or `canvasjs.min.js` ?

File: pj_movie/config/routes.rb [hunk 3]
```diff
@@ -3,6 +3,11 @@
   get 'admin', to: 'admin/top_page#index', as: :admin
 
   namespace :admin do
+    resources :purchases do
+      collection do
+        get 'show'
+      end
+    end
```
Comment (N310) [inline pj_movie/config/routes.rb]: you don't need to use `collection`

File: pj_movie/app/assets/javascripts/admin/script.js (hunk 2 above)
Comment (PechBunsinh) [reply pj_movie/app/assets/javascripts/admin/script.js]: This file is canvasjs.min.js

Comment (N310) [reply pj_movie/app/assets/javascripts/admin/script.js]: you should change this file name to `canvasjs.min.js`

File: pj_movie/app/assets/stylesheets/admin/purchases.scss [hunk 4]
```diff
@@ -0,0 +1,45 @@
+.detail_content{
+  padding: 0px 40px;
+  h1{
+    font-size: 24px;
+    padding: 20px 0;
+  }
+  .info{
+    padding: 20px 0;
+  }
+  #chartContainer{
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/admin/purchases.scss]: Do you use this ID?
I think this is `chartContainerShow`

File: pj_movie/app/views/admin/purchases/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,53 @@
+<div class="detail_content">
+  <div>
+    <h1>Purchase Detail</h1>
+    <div class="info">
+      <p>Video Title:  New Sweet Song</p>
+    </div>
+    <div id="chartContainerShow" style="height: 300px; width: 100%;"></div>
```
Comment (N310) [inline pj_movie/app/views/admin/purchases/show.html.erb]: You should move Style to css file.

**Pull request merged**